packages/react-data-grid/src/CellAction.js

```diff
@@ -52,8 +52,8 @@ class CellAction extends React.Component {
       'cell-action-last': this.props.isFirst
     });
 
-    const actionButtonClasses = classNames('action-button', {
-      'action-button-toggled': this.state.isMenuOpen
+    const actionButtonClasses = classNames('cell-action-button', {
```

Comment:
We have been prefixing class names with rdg. Let's stick to that to avoid clash

`rdg-cell-action-button`

Reply:
ok